recipes/kubernetes_asyncio/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "10.0.0" %}
+
+package:
+  name: kubernetes_asyncio
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/k/kubernetes_asyncio/kubernetes_asyncio-{{ version }}.tar.gz
+  sha256: bcfc3f50605dcdb31a2c05b2627f918e02f3483a8748186a4afce6bb352de21a
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - python-dateutil >=2.5.3
```

Comment:
What is it using `python-dateutil` for during the build?

Reply:
It is not. That is conda-skeleton letf-overs. Should be OK now.

Reply:
Ah ok. Cool. Thanks for clarifying.